app/assets/stylesheets/custom.css.scss

```diff
@@ -1,10 +1,16 @@
-$light-gray: #f7f7f7;
-$white: #fff;
+$primaryColor: #fff;
```

Comment:
Name of variable `primaryColor` should be written in all lowercase letters with hyphens instead of underscores